### DISCUSSION TITLE
Fulltext, especially Felamimail

### DISCUSSION THREAD

**Discussion by lab-at-nohl** [tine20/tine20#7389] [Q&A] [answered]
How does it work? Can I make use of IMAP search, if it provides a fast Index?

Btw, for what else is Fulltext implemented?

**Comment by pschuele** [comment, marked as answer]
hi,
we do not support fulltext indexing in Felamimail, yet.

fulltext is used to quickly search in description fields (for example of contacts) and to search for file contents in the Filemanager.
for fulltext indexing of files, you need apache tika (it is already installed and configured in the docker images).

in the docs: https://tine-docu.s3web.rz1.metaways.net/admins/howto/tine20AdminFilemanagerFilesystem/#volltextindizierung-mit-apache-tika (in german)

**Reply by pschuele** [reply]
there also is a feature request for Felamimail fulltext indexing of mail bodies: https://github.com/tine20/tine20/issues/669
